demo_umami_content/src/InstallHelper.php

```diff
@@ -275,7 +287,7 @@ public function importBlockContent() {
           'uri' => 'internal:' . call_user_func(function () {
             $nodes = $this->entityTypeManager->getStorage('node')->loadByProperties(['title' => 'Super easy vegetarian pasta bake']);
             $node = reset($nodes);
-            return $node->toUrl()->toString();
+            return $this->aliasManager->getAliasByPath('/node/' . $node->id());
```

Comment:
@eli-t Can the following be a good code ?

`'uri' => 'internal:/recipes/super-easy-vegetarian-pasta-bake',
'title' => 'Super easy vegetarian pasta bake',`